test/Dommel.IntegrationTests/LikeTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace Dommel.IntegrationTests
```

Comment:
Can you put this test into the `Dommel.Tests` project? You're not using a database here so it's just a regular unit test.

Reply:
@samuelpampolini can you take a look at this? You can just move the file to the Dommel.Tests project.